certification/errors/errors.go

```diff
@@ -31,11 +31,12 @@ var (
 	ErrUnsupportedGoType               = errors.New("go type unsupported")
 	ErrSaveFileFailed                  = errors.New("failed to save file to artifacts directory")
 	ErrNon200StatusCode                = errors.New("error calling remote API")
-	Err409StatusCode                   = errors.New("remote API returned conflict")
+	ErrPyxis409StatusCode              = errors.New("remote API returned conflict")
```

Comment:
Non Blocking: I get the change of this for clarity, but if `preflight` ever has to call another backend service, having the more generic error might make sense.

Reply:
nit: .. or a more specific message like "pyxis remote API returned conflict" (in case we would have ErrOtherExtService409StatusCode in the future)

Reply:
I meant to change the message to include a note about Pyxis. I think @acornett21 has a valid point, though, so if we decide to add more remote API calls, I would vote instead to have a generic HTTP Status Code that gets wrapped with the status code itself (so we can reuse it anywhere and report an error as we need to). This will be fixed.

Reply:
This is resolved. I'll let @acornett21 resolve the comment thread when you have time.

Reply:
This makes more sense now, and I agree if the time comes where we call other systems, we wrap a generic message.